fastlane/metadata/fr-FR/description.txt

```diff
@@ -0,0 +1 @@
+ScanNow est une application simple et rapide pour numériser des documents sur iOS. Avec ScanNow, vous pouvez numériser, enregistrer et partager n'importe quel document en quelques secondes. ScanNow utilise la caméra de votre iPhone pour prendre des photos de vos documents et les convertit en images ou en PDF. Vous pouvez ensuite partager vos documents numérisés par e-mail, SMS, ou les enregistrer sur votre appareil pour une utilisation ultérieure. ScanNow est parfait pour numériser des reçus, des notes, des factures, des cartes de visite, des certificats, etc. Téléchargez ScanNow aujourd'hui et commencez à numériser!
```

Comment:
Here should be translated to French content of `fastlane/metadata/en-GB/description.txt`

Reply:
No. It's not translated to French